lib/medusa.ex

```diff
@@ -50,14 +59,34 @@ defmodule Medusa do
     Medusa.Broker.new_route(route, function, opts)
   end
 
+  defp is_message_valid?(event, payload, metadata) do
```

Comment:
might be wrong, but I think elixir style is to stick private functions at the bottom of the module?